src/sql-parser/src/ast/defs/name.rs

```diff
@@ -333,6 +334,9 @@ pub enum IdentError {
         suffix: String,
         attempts: usize,
     },
+
+    #[error("invalid identifier: {}", .0.quoted())]
+    WouldCausePathTraversalInUrl(String),
```

Comment:
To make this more generic, maybe we rename this to just `Invalid` as opposed to having individual errors for each case? Up to you though, it's easy to change this in the future!

Reply:
Done.